services/gateway/apis/http-version3/swagger/definitions/nft.json

```diff
@@ -124,5 +124,40 @@
                 "type": "object"
             }
         }
+    },
+    "nftSupportedWithEnvelope": {
+        "type": "object",
+        "required": [
+            "data",
+            "meta"
+        ],
+        "properties": {
+            "data": {
+                "description": "NFT module supported collection IDs information.",
+                "type": "object",
+                "required": [
+                    "isSupportAllNFTs",
+                    "exactCollectionIDs"
+                ],
+                "properties": {
+                    "isSupportAllNFTs": {
```

Comment:
Do we not need a pattern specification similar to `patternTokenIDs` as in [`/token/summary`](https://petstore.swagger.io/?url=https://betanet-service.lisk.com/api/v3/spec#/Token/get_token_summary)?

Reply:
Applied.